exchanges/exchange.go

```diff
@@ -549,94 +553,148 @@ func (b *Base) SetPairs(pairs currency.Pairs, assetType asset.Item, enabled bool
 		return err
 	}
 
-	var newPairs currency.Pairs
 	for x := range pairs {
-		newPairs = append(newPairs, pairs[x].Format(pairFmt.Delimiter,
-			pairFmt.Uppercase))
+		pairs[x] = pairs[x].Format(pairFmt)
 	}
 
-	b.CurrencyPairs.StorePairs(assetType, newPairs, enabled)
-	b.Config.CurrencyPairs.StorePairs(assetType, newPairs, enabled)
+	b.CurrencyPairs.StorePairs(assetType, pairs, enabled)
+	b.Config.CurrencyPairs.StorePairs(assetType, pairs, enabled)
 	return nil
 }
 
 // UpdatePairs updates the exchange currency pairs for either enabledPairs or
 // availablePairs
-func (b *Base) UpdatePairs(exchangeProducts currency.Pairs, assetType asset.Item, enabled, force bool) error {
-	exchangeProducts = exchangeProducts.Upper()
-	var products currency.Pairs
-	for x := range exchangeProducts {
-		if exchangeProducts[x].String() == "" {
-			continue
+func (b *Base) UpdatePairs(incomingPairs currency.Pairs, assetType asset.Item, enabled, force bool) error {
```

Comment:
Is there potential backwards compatability issues? I am just unsure, the intention of this update was to force config formatting so that when unmarshalling occurs for example when saved without delimiter just uppercasing 'QTUMETH' it will then be loaded into memory as QTU-METH and if request formatting is applied to the currency pair as just upper casing then it doesn't matter but if the exchange wants a specific delimiter then it could get funky.